code/controllers/configuration.dm

```diff
@@ -746,6 +737,18 @@
 					config.enable_gamemode_player_limit = 1
 				if("byond_account_age_threshold")
 					config.byond_account_age_threshold = text2num(value)
+				// Discord stuff
+				if("enable_discord_webhooks")
+					discord_webhooks_enabled = TRUE
+				if("discord_webhooks_admin_role_id")
+					discord_admin_role_id = "[value]" // This MUST be a string because BYOND doesnt like massive integers
```

Comment:
isn't `value` a string anyway? So just this would work:
```suggestion
					discord_admin_role_id = value // This MUST be a string because BYOND doesnt like massive integers
```

Reply:
Better safe than sorry incase config parsing is ever changed without it being fully thought through (Or as an internal BYOND update which breaks everything)

Reply:
If either changes then a whooooole lot more will fail as well. Not just the discord integration

Reply:
Im keeping the safety check in, because it also makes it very clear to anyone looking at it that the value is a string. Someone can quite easily forget this is a string and put an int in its place because IDs are numbers, then break everything

Reply:
It's more that `value` is always a string. You'll need to parse it to a number like on line 739. Only time this can break is when `file2list` suddenly started to return non strings.